src/components/ContactUsForm/Contact_Us.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./Contact_Us.css";
+
+const ContactUs = () => {

```

Comment:
this actually doesnt match the figma design, https://www.figma.com/design/kHauIvxVeOb8diy8jO6jJd/Website?node-id=124-5&t=KYWacSRJGK7iiUoH-0

We can go in such manner

<label> Full Name </label>
<input />

 <label> Email </label>
<input />